linux_os/guide/system/accounts/accounts-pam/locking_out_password_attempts/accounts_passwords_pam_faillock_deny/bash/shared.sh

```diff
@@ -2,4 +2,33 @@
 
 {{{ bash_instantiate_variables("var_accounts_passwords_pam_faillock_deny") }}}
 
-{{{ bash_set_faillock_option("deny", "$var_accounts_passwords_pam_faillock_deny") }}}
+SYSTEM_AUTH="/etc/pam.d/system-auth"
+PASSWORD_AUTH="/etc/pam.d/password-auth"
+FAILLOCK_CONF="/etc/security/faillock.conf"
+
+if [ $(grep -c "^\s*auth.*pam_unix.so" $SYSTEM_AUTH) > 1 ] || \
+   [ $(grep -c "^\s*auth.*pam_unix.so" $PASSWORD_AUTH) > 1 ]; then
+   echo "Skipping remediation because there are more pam_unix.so entries than expected."
+   false
+fi
+
+if [ -f $FAILLOCK_CONF ]; then
```

Comment:
So is the `/etc/security/faillock.conf` present also on RHEL7? If not, would it make sense to have different remediation for RHEL7?

Reply:
Hum, it is one approach to create two remediation. However, the first versions of RHEL8 also didn't have `/etc/security/faillock.conf`. So, part of the logic used for RHEL7 is also used for first versions of RHEL8. In this context, looks reasonable to have one single remediation with less duplicated code. Do you agree?